### ISSUE TITLE
Cannot GET /servers and Cannot GET /me

### ISSUE THREAD

**Issue by canopus310** [caelinj/discord.js-dashboard#9]
ive gotten everything setup its just those two errors I get while using the dashboard. I'm not sure if I've done something wrong.

**Comment by nathan-dev27** [comment]
because /servers and /me isnt set up if u want to get it done code it urself

**Comment by charfweh** [comment]
yes that's right, it hasn't been implemented, but you can try follow this https://github.com/charfweh/discord-bot-dashboard, its a forked and updated version of this very repository, hope it helps.
